packages/x-components/src/design-system/components/tag/default.scss

```diff
@@ -53,6 +54,19 @@
     --x-size-padding-left-suggestion-default: var(--x-size-padding-left-tag-default);
     --x-size-gap-suggestion-default: var(--x-size-gap-tag-default);
 
+    // border
+    --x-size-border-width-suggestion-default: var(--x-size-border-width-tag-default);
+    --x-size-border-width-top-suggestion-default: var(--x-size-border-width-tag-default);
+    --x-size-border-width-right-suggestion-default: var(--x-size-border-width-tag-default);
+    --x-size-border-width-bottom-suggestion-default: var(--x-size-border-width-tag-default);
+    --x-size-border-width-left-suggestion-default: var(--x-size-border-width-tag-default);
+
+    --x-size-border-radius-suggestion-default: var(--x-size-border-radius-tag-default);
+    --x-size-border-radius-bottom-right-suggestion-default: var(--x-size-border-radius-bottom-right-tag-default);
+    --x-size-border-radius-bottom-left-suggestion-default: var(--x-size-border-radius-bottom-leftt-tag-default);
+    --x-size-border-radius-top-right-suggestion-default: var(--x-size-border-radius-top-right-tag-default);
+    --x-size-border-radius-top-left-suggestion-default: var(--x-size-border-radius-top-left-tag-default);
```

Comment:
all these tokens must be also in the `suggestion/default.tokens.scss`

Reply:
they are already there